port/osx/omrosbacktrace_impl.c

```diff
@@ -429,4 +438,9 @@ omrintrospect_backtrace_symbols_raw(OMRPortLibrary *portLibrary, J9PlatformThrea
 	}
 
 	return i;
+#elif defined(OMR_ARCH_AARCH64) /* defined(OMR_ARCH_X86) */
+	return 0;
+#else
```

Comment:
```suggestion
#else  /* defined(OMR_ARCH_AARCH64) */
```

Reply:
nit: but just one space after `else`

Reply:
and the comment should match the `#if` expression (`/* defined(OMR_ARCH_X86) */`).

Reply:
as per the [coding standard](https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif), 
- `#elif` and `#else` should have comments that echo the previous condition.
- `#endif` should have a comment describing the matching `#if`, `#ifdef` or `#ifndef`, where practical.

**Example:**
```
#if defined(A) || defined(B)
...
#elif !defined(C) /* defined(A) || defined(B) */
...
#else /* !defined(C) */
...
#endif /* defined(A) || defined(B) */
```
So, the `#else` comment should have the previous `#elif` condition which is `defined(OMR_ARCH_AARCH64)`.